### PR TITLE
hh_headed

### PR DESCRIPTION
**Pull request by Xian152**
generate hh_headed for all household members, rather than just for the household head

### CODE REVIEW

File: 15_household.do [hunk 1]
```diff
@@ -7,12 +7,13 @@
     clonevar hh_id = hhid
 	
 *hh_headed	Head's highest educational attainment (1 = none, 2 = primary, 3 = lower sec or higher)
-    recode hv106 (0 = 1) (1 = 2) (2/3 = 3) (8=.) if hv101 == 1,gen(hh_headed)
-    
-    gen name = "`name'"
```
Comment (wengxyu1030) [inline 15_household.do:12]: Should you keep this line? If you are using country-specific code in the following command should you keep the name variable generation?

File: 15_household.do [hunk 2]
```diff
@@ -7,12 +7,13 @@
     clonevar hh_id = hhid
 	
 *hh_headed	Head's highest educational attainment (1 = none, 2 = primary, 3 = lower sec or higher)
-    recode hv106 (0 = 1) (1 = 2) (2/3 = 3) (8=.) if hv101 == 1,gen(hh_headed)
-    
-    gen name = "`name'"
+    recode hv106 (0 = 1) (1 = 2) (2/3 = 3) (8 9=.) if hv101 == 1,gen(headed)
+	bysort hh_id: egen hh_headed = mean(headed)
```
Comment (wengxyu1030) [inline 15_household.do:11]: Can you clarify why doing this? You saw multiple household head in one family so you want to take the mean? If so I think we are not generating statistics here however producing raw data, compress at the household level might be questionable.

Comment (Xian152) [reply 15_household.do:11]: The most sensible way is to use min(headed)
I use mean just to help me to identify the survey if there exist households assigned with more than 1 hh head who have different education levels. Turns out none of the surveys have that. I will change it back to min(headed) later.

File: 15_household.do (hunk 1 above)
Comment (Xian152) [reply 15_household.do:12]: Ohh, yes you are right. I need to keep this line to make the code work for Yemen.

**Pull request closed without merging**